### DISCUSSION TITLE
preset help

### DISCUSSION THREAD

**Discussion by kalle07** [mcmonkeyprojects/SwarmUI#363] [Q&A]
so i created a preset for sdxl

and i only choose a resolution, a sampler
and an fixed negativ prompt

if i start swarmui
i choose an sdxl model
klick on the preset and press "apply preset"

and now every time i generate an image on the same seed its the the same image no matter i wrote in the positiv prompt ?!?
but in image meta-data there is the right prompt i wrote but the same image content...
seem i shoot some options but i dont know how?
it give an reset for all options, or can i delete a file to reset to standard ?

also if i apply another preset some functions are still enabeld from the first preset
like clip models 
(i think all options should be allways set in standard, unless i changed it in the preset)

**Comment by mcmonkey4eva** [comment]
You can reset to default with this button:

![image](https://github.com/user-attachments/assets/8900eab4-87f9-4e37-949b-244e89ffca6a)

As to your prompt getting stuck... that doesn't make sense, I'd need to see example output images to look at the metadata and see what's up. Possibly you applied an empty prompt or something in the preset and left it selected and not applied?

**Reply by kalle07** [reply]
i see ...

i had the problem i discribed and it depends on that ... that the if one preset applied and i set eg a clip-model or i set cfg=1 ... and i apply a new preset these two options are unchanged and not reset to standard.
so if i have only one of 10 presets that changes one paramete like cfg=1, i must set it it in a all other presets to standard or any other number, you see what i mean ?
